src/pages/Options/Options.tsx

```diff
@@ -1,12 +1,53 @@
-import React from 'react';
-import './Options.css';
+import UserReactors from 'components/Options/UserReactors';
 
-interface Props {
-  title: string;
-}
+import ReactorSearch from 'components/Options/ReactorSearch';
+import { Box, Center, VStack } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
+import { Channel } from 'services/api';
 
-const Options: React.FC<Props> = ({ title }: Props) => {
-  return <div className="OptionsContainer">{title} Page</div>;
+const Options = () => {
+  const [userReactors, setUserReactors] = useState<Channel[]>([]);
+
+  useEffect(() => {
+    chrome.storage.local.get(['userReactors'], function (result) {
+      if (!result.userReactors) return;
+      setUserReactors(result.userReactors);
+    });
+  }, []);
+
+  const setReactors = (channels: Channel[]) => {
+    setUserReactors(channels);
+    chrome.storage.local.set({ userReactors: channels }, function () {
+      console.log(chrome.runtime.lastError);
+    });
+  };
+  const addReactor = (c: Channel) => {
+    setReactors([...userReactors, c]);
+  };
+
+  const deleteReactor = (id: string) => {
+    const deleted = userReactors.filter((reactor) => reactor.id !== id);
+    setReactors(deleted);
+  };
+
+  return (
+    <Center>
```

Comment:
Have thoughts on how to explain this option's behavior to the user?

Reply:
Yeah I was wondering this too, since our options page is pretty bare. Right now it's just the placeholder in the search bar... Design wise and ux wise, I don't really have any ideas yet. I'm open to suggestions